src/wasm-linker.h

```diff
@@ -44,6 +46,26 @@ class LinkerObject {
     Relocation(Kind kind, uint32_t* data, Name symbol, int addend) :
         kind(kind), data(data), symbol(symbol), addend(addend) {}
   };
+  // Information about symbols
+  struct SymbolInfo {
+    std::unordered_set<cashew::IString> implementedFunctions;
```

Comment:
They actually can't be because it's `IString` and not `Name` that has `std::hash` defined for it. Originally I just thought it better to use IString as the key but now the hash maps are starting to proliferate, maybe I should revisit that. Anyway, I'd like to make that a separate PR.